### PR TITLE
Tonyvanswet/fix todos part2

### PR DESCRIPTION
**Pull request by tonypls**

### CODE REVIEW

File: web/src/features/map/map-utils/generateTopos.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export interface Topo {
 /**
  * This function takes the topojson file and converts it to a geojson file
  */
-const generateTopos = (theme: Theme, spatialAggregate: string): MapGeometries => {
+const generateTopos = (theme: any, spatialAggregate: string): MapGeometries => {
```
Comment (Kongkille) [inline web/src/features/map/map-utils/generateTopos.ts:31]: Why are we changing to any here?

**Review by Kongkille — APPROVED** [review]
Looks good, do we need to lose theme typing?

File: web/src/features/map/map-utils/generateTopos.ts (hunk 1 above)
Comment (tonypls) [reply web/src/features/map/map-utils/generateTopos.ts:31]: I believe it was incorrectly typed a while ago and then this has been referencing a non existent type for the last few weeks

Comment (tonypls) [reply web/src/features/map/map-utils/generateTopos.ts:31]: hmm looks like it was just renamed to MapTheme and never updated elsewhere. I'll fix that now :D

**Pull request merged**